Sources/StreamChatUI/ChatMessageList/ChatMessageListRouter.swift

```diff
@@ -42,4 +42,13 @@ open class _ChatMessageListRouter<ExtraData: ExtraDataTypes>: ChatRouter<_ChatMe
         let navigation = UINavigationController(rootViewController: preview)
         rootViewController.present(navigation, animated: true)
     }
+    
+    open func showImageGallery(for message: _ChatMessage<ExtraData>) {
+        let imageGalleryVC = _ImageGalleryVC<ExtraData>()
+        imageGalleryVC.modalPresentationStyle = .overFullScreen
+        imageGalleryVC.content = message
+        
+        // TODO: Animated transition?
+        rootViewController.present(imageGalleryVC, animated: true)
```

Comment:
images are usually enlarged to the image gallery -> do we want that, too? My suggestion is to probably do that in some next iteration

Reply:
I think it's OK to do it now. We already have a similar thing in the message actions popover. It's important to have it because we for sure want to the interaction where you can swipe the image to close it.